### PR TITLE
New feature Capture Screen

### PR DESCRIPTION
**Pull request by 26tanishabanik**
Added new feature Capture Screen. Issue https://github.com/sukeesh/Jarvis/issues/894

### CODE REVIEW

**Comment by ShreyPatel4** [comment]
this is great work.

suggestion:- if we first just made the textual version then implement for voice models.?

File: jarviscli/plugins/screencapture.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import platform
+import pyautogui
+
+
+@alias("capture", "screen")
+@plugin('screen_capture')
+def capture_screen(jarvis, s):
+
+    if (platform.system() == 'Linux'):
+        proc = subprocess.Popen(['dpkg',
```
Comment (pnhofmann) [inline jarviscli/plugins/screencapture.py:14]: Hi!
good thing to handle these kind of exceptions. We should also make the installer aware of scrot / pyautogui.

But dpkg is only for ubuntu or debian. The way intended for these kind of checks is:

```
@require(native='scrot')
@plugin('screen_capture')
```

**Comment by 26tanishabanik** [comment]
> this is great work.
> 
> suggestion:- if we first just made the textual version then implement for voice models.?

@ShreyPatel4  I think it applies to both

**Comment by pnhofmann** [comment]
Thank you!

**Pull request merged**